css/templates/travel/elements/_border.css

```diff
@@ -0,0 +1,24 @@
+/**
+  * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS-IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+/*
+ * border
+ *
+ */
+
+ .travel-border-gray {
```

Comment:
do not create new files for one of classes move them to main page css.

Reply:
i think this might be a merge carryover

Reply:
No, we haven't merged this file yet. We use this class in results page.